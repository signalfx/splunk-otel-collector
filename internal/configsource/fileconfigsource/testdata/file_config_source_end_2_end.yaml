config_sources:
  file:

config:
  scalar_data: ${file:./testdata/scalar_data_file}
  yaml_data: $file:./testdata/yaml_data_file
