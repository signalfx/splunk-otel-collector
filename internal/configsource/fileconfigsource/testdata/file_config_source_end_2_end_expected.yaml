config:
  scalar_data: 42
  yaml_data:
    field: value
    map:
      k0: 42
      k1: v1
